docs/articles/test/figures.html

```diff
@@ -110,35 +110,17 @@ <h1>Test: Figures</h1>
             
       
       <small>Source: <a href="https://github.com/hadley/pkgdown/blob/master/vignettes/test/figures.Rmd"><code>vignettes/test/figures.Rmd</code></a></small>
-      <p class="hidden name"><code>figures.Rmd</code></p>
+      <div class="hidden name"><code>figures.Rmd</code></div>
 
     </div>
 
     
     
-<div id="regular" class="section level2">
-<h2 class="hasAnchor">
-<a href="#regular" class="anchor"></a>Regular</h2>
 <div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">plot</span>(<span class="dv">1</span><span class="op">:</span><span class="dv">10</span>)</a></code></pre></div>
```

Comment:
Do you have the latest source version of this file?

Reply:
I am rebased and the version I have is the same [in the repo](https://raw.githubusercontent.com/r-lib/pkgdown/master/vignettes/test/figures.Rmd). Not sure where the "Square" section came from. Maybe you forgot to push the source, but did push the docs version?

Reply:
@hadly don't forget to de-reset this file if you can find it.